src/components/interactive/Button.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import baseElement from '../../util/baseElement';
+
+const Button = ({
+  theme,
+  isLoading,
+  iconPlacement,
+  Icon,
+  Link,
+  disabled,
+  children,
+  isDestructive,
+  block,
+  ...rest
+}) => {
+  const Element = baseElement({ href: rest.href, as: Link });
+  return (
+    <Element
+      {...rest}
+      className={cx(
+        'fdsButton',
+        'fontStyle--caps',
+        'fontWeight--bold',
+        'display--inlineFlex',
+        'rounded--all',
+        'fontSize--m',
+        'alignChild--center--center',
+        {
+          'fdsButton--blue': theme === 'blue',
+          'fdsButton--outlined': theme === 'outlined',
+          'fdsButton--ghost': theme === 'ghost',
+          'fdsButton--isDestructive': isDestructive,
+          'fdsButton--loading': isLoading,
+          'fdsButton--disabled': disabled,
+          'fdsButton--block': block,
+        }
+      )}
+      disabled={disabled && Element === 'button'}
+    >
+      <span className={cx({ 'fdsButton--hidden': isLoading })}>{children}</span>
+      {Icon && (
+        <div
+          className={cx('fdsButton-icon', {
+            'fdsButton-icon--left': iconPlacement === 'left',
+            'margin--left--half': iconPlacement === 'right',
+            'margin--right--half': iconPlacement === 'left',
+            'fdsButton--hidden': isLoading,
+          })}
+        >
+          <Icon size="xs" />
+        </div>
+      )}
+    </Element>
+  );
+};
+
+Button.defaultProps = {
+  theme: 'primary--blue',
```

Comment:
Need to change this default prop to reflect new theming options

Reply:
Gooood catch.